nanshe_ipython.ipynb

```diff
@@ -713,6 +713,17 @@
         "\n",
         "        imgs_min, imgs_max = [s.result() for s in status]\n",
         "\n",
+        "    shifts = LazyZarrDataset(fn, \"shifts\")\n",
+        "\n",
+        "    fig, axs = plt.subplots(nrows=shifts.shape[1], sharex=True)\n",
+        "    fig.subplots_adjust(hspace=0.0)\n",
+        "    for i in range(shifts.shape[1]):\n",
+        "        axs[i].plot(shifts[:, i][...])\n",
+        "        axs[i].set_ylabel(\"%s (px)\" % chr(ord(\"X\") + shifts.shape[1] - i - 1))\n",
+        "        [v.set_linewidth(2) for v in axs[i].spines.values()]\n",
+        "    axs[-1].set_xlabel(\"Frame (#)\")\n",
+        "    axs[-1].set_xlim((0, shifts.shape[0] - 1))\n",
```

Comment:
Label X-axis and restrict range to the frames included so it doesn't go negative or out past the number of frames. By default Matplotlib tries to do things like this.